src/Gui/ViewProviderLink.cpp

```diff
@@ -2893,7 +2893,7 @@ void ViewProviderLink::setEditViewer(Gui::View3DInventorViewer* viewer, int ModN
             dragger->setUpAutoScale(viewer->getSoRenderManager()->getCamera());
             viewer->setupEditingRoot(pcDragger,&dragCtx->preTransform);
 
-            auto task = new TaskCSysDragger(this, dragger);
+            auto task = new TaskCSysDragger(nullptr, dragger);
```

Comment:
This will cause a segmentation fault. The constructor of TaskCSysDragger forwards its first argument to TaskTransform and this constructor will access it without checking for null.

Reply:
Yes, this is temporary until I refactor `ViewProviderLink`.